azure-keyvault/HISTORY.rst

```diff
@@ -2,6 +2,27 @@
 
 Release History
 ===============
+1.0.0b1 (2018-04-10)
+++++++++++++++++++++
+
+* Upgraded to autorest 3.0 generated code
```

Comment:
I used to copy the "General Breaking Changes" from here, to be explicit:
https://github.com/Azure/azure-sdk-for-python/blob/master/azure-sdk-tools/changelog_generics.rst